Instructions in code-setup fails to build on darwin

## Description 

The code setup and build instructions in [code-setup doc](https://github.com/ChakshuGautam/cQube-ingestion/blob/master/docs/01-code-setup.md) fails to build on `darwin`. 

#### Steps to reproduce: 
 - Follow the build instructions on code-setup doc on a `darwin` machine.


#### Root Cause:
 - There seems to be an [explicit dependency](https://github.com/ChakshuGautam/cQube-ingestion/blob/6892fa499c002bfef3f3943b0786bccfef2fba79/impl/c-qube/package.json#L47) on the linux-x64 binary for node-polars npm dependency which causes the the npm install to look for os type "linux".  
 
 `npm install` Fails with the error. (`yarn install` fails with a similar error) - 
 
 ``` 
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for nodejs-polars-linux-x64-gnu@0.7.2: wanted {"os":"linux","arch":"x64"} (current: {"os":"darwin","arch":"arm64"})
npm ERR! notsup Valid OS:    linux
npm ERR! notsup Valid Arch:  x64
npm ERR! notsup Actual OS:   darwin
npm ERR! notsup Actual Arch: arm64
 ```

#### Potential Fix: 

The build succeeded after removing the mentioned dependency from `package.json`.  This was verified by successfully  running the tests via `yarn test` which succeeded after removing the dependency.

There already seems to be a dependency on [nodejs-polars](https://github.com/ChakshuGautam/cQube-ingestion/blob/6892fa499c002bfef3f3943b0786bccfef2fba79/impl/c-qube/package.json#L46), which makes me think that explicit dependency on the linux-x64 binary for this package can be removed. 

#### Additional Info: 
 - Tested on a MacOS 12.5.1 running Apple Silicon (M1)

## Goals

The goal is to fix the issue as outlined in the description either via the suggested fix or some other way.

## Project

cQube-ingestion

## Organisation Name
Samagra | Transforming Governance

## Mentor(s)
@techsavvyash @ChakshuGautam 


## Technical Skills Needed

NodeJS, NPM

## Complexity

Low

## Category

Bug fix

## Sub Category

Project Setup

## Domain

Others

--- a/impl/c-qube/package.json
+++ b/impl/c-qube/package.json
@@ -44,7 +44,6 @@
     "lodash": "^4.17.21",
     "nestjs-pino": "^3.1.2",
     "nodejs-polars": "^0.7.2",
-    "nodejs-polars-linux-x64-gnu": "^0.7.2",
     "pg": "^8.10.0",
     "picocolors": "^1.0.0",
     "pino-http": "^8.3.3",

